src/commands/paas/publish.ts

```diff
@@ -30,38 +26,50 @@ class PaasPublish extends Kommand {
   ]
 
   async runSafe() {
-    const paas = new KuzzleSDK({
-      protocol: 'ws',
-      host: this.host,
-      port: this.port,
-      ssl: this.ssl,
-      apiKey: this.flags.token,
-    })
-
-    try {
-      await paas.init(this);
-
-      const user = await paas.auth.getCurrentUser();
-      this.logInfo(`Logged as "${user._id}" for project "${this.args.project}`);
-
-      const [image, tag] = this.args.image.split(':');
-      this.logInfo(`Deploy application with image "${image}:${tag}"`)
-
-      await paas.query({
-        controller: 'github',
-        action: 'updateImage',
-        body: {
-          namespace: this.args.project,
-          image,
-          tag,
-        }
-      });
-
-      this.logOk('Deployment in progress');
-    }
-    catch (error: any) {
-      this.logKo(error.message)
+    const apiKey = await this.getCredentials();
+
+    await this.initPaasClient({ apiKey });
+
+    const user = await this.paas.auth.getCurrentUser();
+    this.logInfo(`Logged as "${user._id}" for project "${this.args.project}"`);
+
+    const [image, tag] = this.args.image.split(':');
+    this.logInfo(`Deploy application with image "${image}:${tag}"`)
+
+    await this.paas.query({
+      controller: 'github',
```

Comment:
This will soon be updated to `config`

```suggestion
      controller: 'config',
```